orbstation/code/antagonists/pod_infiltrators/disaster_survivor.dm

```diff
@@ -0,0 +1,52 @@
+// Not an antagonist, but a "normal" infiltrator. Mostly for plausible deniability purposes, but you can have some fun.
+
+/datum/job/disaster_survivor
+	title = "Disaster Survivor"
+
+/datum/round_event_control/disaster_survivor
+	name = "Disaster Survivor"
+	description = "A desperate survivor who spawns in a pod, and must do everything they can to stay alive."
+	category = EVENT_CATEGORY_FRIENDLY //they might not be helpful but they aren't an antag
+	typepath = /datum/round_event/ghost_role/disaster_survivor
+	weight = 8 // same weight as the other infiltrators
```

Comment:
I have actually been wondering, why _do_ we have random event support for the changeling/traitor infiltrators?
They're controlled by dynamic, as far as I'm aware there isn't any reason to _also_ have a random event except for backwards compatibility.